Feature_slicing_Operations.py

```diff
@@ -0,0 +1,22 @@
+#Program should accept
```

Comment:
I don't understand what this program is doing. Please make changes according the instructions given.

<img width="340" alt="image" src="https://user-images.githubusercontent.com/38807024/200740503-2602988e-2e38-4405-836d-de70d0f19fde.png">